lms/djangoapps/discussion_api/api.py

```diff
@@ -110,6 +110,27 @@ def _is_user_author_or_privileged(cc_content, context):
     )
 
 
+def _update_vote_count(voted, serializer):
+    """
+    Updates the vote_count in the given serializer
+
+    If the user has already voted and `voted == True` do nothing,
+    else +1 to the vote count.
+    If the user has not voted and `voted == False` do nothing,
+    else -1 to the vote count.
```

Comment:
make this logic more clear? the "if...else" was confusing at first, maybe you could simplify to  
If the user has already voted and `voted == False` then +1 to the vote count